ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -2204,6 +2204,12 @@ def test_severity(self):
             data = json.loads(result.stdout_text)
             assert len(data) == 0
 
+            cmd = ["ipa-healthcheck", "--severity", severity,
+                   "--failures-only", "--output-type", "human"]
+            result = self.master.run_command(cmd)
+            assert result.stdout_text == "No issues found.\n"
```

Comment:
I think this is a better fit as a separate test. I think I'd rename test_human_output to test_human_severity and stick your code into test_human_output and drop the severity part.

You'd also be able to use run_healthcheck(output_type="human", failures_only=True)

Reply:
Thank you, Rob. I _think_ I understood, changes pushed.